source/09-audit/03-timelines.rst

```diff
@@ -0,0 +1,59 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+Audit Timelines
+---------------
+
+An audit is valid for twelve months from the date of seeding: that is, an audit seeded in April 2025 will be valid until the end of March 2026.
+
+The audit process can be lengthy and report providers should take this into consideration when planning for audits. A typical timeline is shown in table 9.a below.
+
+Table 9.a (below): Audit Timelines
+
+.. only:: latex
+
+   .. tabularcolumns:: |>{\raggedright\arraybackslash}\Y{0.13}|>{\raggedright\arraybackslash}\Y{0.17}|>{\parskip=\tparskip}\Y{0.37}|>{\raggedright\arraybackslash}\Y{0.33}|
+
+.. list-table::
+   :class: longtable
+   :widths: 38 38 24
+   :header-rows: 1
+
+   * - Process
+     - Usual Duration
+     - Example
+
+   * - Seeding
+     - 1 calendar month
+     - February
+
+   * - Usage processed, reports available
+     - Up to 28 days from end of seeding month
+     - March
+
+   * - Report reconciliation
+     - 4 to 6 weeks
+     - April
+
+   * - Fixes after interim report
+     - Up to 3 months
+     - May to July
+
+   * - Report reconciliation after fixes
+     - 1 month
+     - August
+
+\*Report providers may request extensions to the usual timelines under the following circumstances.
+
+* **Delayed seeding** - where a report provider is able to demonstrate that they have run the COUNTER Validation Tool and are in the process of fixing problems identified by the Tool before commencing an audit, they may apply for up to three months delay in the seeding process.
+* **Extension to the fix period** - where an Interim Report has identified significant problems with COUNTER reports, the report provider may apply for an extension of the fix period from three to up to six months.
+
+Applications for extensions MUST be sent to both the Project Director and the auditor, and will be considered on a case-by-case basis. Only one extension of each type will be granted for any audit, and all extensions will be tracked in the Registry.
+
+
+Alternate year audits
+"""""""""""""""""""""
+
+Some report providers are eligible to be audited once every 24 months. Specifically, report providers delivering COUNTER reports for a single platform, where that platform includes up to 150 books OR 15 journals OR one database, may apply to the Project Director for alternate year audit status. Report providers with historic permission to be audited in alternate years will also be considered but they MUST reapply to the Project Director for confirmation.
```

Comment:
Does "history permission" only refer to the currently existing permissions or also to permissions granted according to this rule (i.e. would a new application be required for each audit cycle)?